src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<String, Integer> storageFruits;
+
+    public Storage() {
+        storageFruits = new HashMap<>();
+    }
+
+    public Map<String, Integer> getStorageFruits() {
+        return storageFruits;
+    }

```

Comment:
If you use Map collection in your code your should override equals and hashCode methods to ensure correct data processing.